contracts/Liquidation.sol

```diff
@@ -45,6 +47,9 @@ contract Liquidation is ILiquidation, Ownable {
         uint256 timestamp
     );
     event InvalidClaimOrder(uint256 indexed receiptId);
+    event ReleaseTimeUpdated(address indexed owner, uint256 releaseTime);
+    event MinimumLeftoverGasCostMultiplierUpdated(address indexed owner, uint256 minimumLeftoverGasCostMultiplier);
+    event MaxSlippageUpdated(address indexed owner, uint256 maxSlippage);
```

Comment:
These should all have tests associated with them; i.e. for the setters that emit these events, we should have a test checking they emit them (`expect(...).to.emit(...).withArgs(...)`)